swaywm/wlroots
# Implement xwayland primary selection sync

Test plan:
* Check that copying between wayland and xwayland clients still works
* Select some text in wayland, press middle button in xwayland
* Same but swap wayland and xwayland

Fixes #513
